.github/workflows/release_and_publish.yml

```diff
@@ -0,0 +1,44 @@
+# Release package on GitHub and publish to PyPI
+# IMPORTANT -- 1 MANUAL STEP
+#   * FOLLOWING TAGGED RELEASE
+#       - update CHANGELOG.md
+#--------------------------------------------------
+
+name: Build, Release, and publish
+
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Manual Release
+        default: test
+        required: false
+
+jobs:
+  build:
+    name: Create release & publish to PyPI
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Set up python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.x"
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools wheel twine jupyter urllib3 pandas pyyaml
+          python setup.py sdist bdist_wheel
+
+      - name: Publish distribution 📦 to PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          user: __token__
+          password: ${{ secrets.PYPI_PASSWORD }}
```

Comment:
Have you added the PyPI token as a repository secret?

Reply:
Yes, just added it.